src/components/Select/Select.tsx

```diff
@@ -0,0 +1,115 @@
+import * as React from "react";
+import bem from "../../utils/bem";
+import Icon from "../Icons/Icon";
+
+export interface SelectProps {
+    /** blockName for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    blockName?: string;
+
+    /** Modifiers array for use with BEM. See how to work with modifiers and BEM here: http://getbem.com/introduction/ */
+    modifiers?: string[];
+
+    /** ID that other components can cross reference for accessibility purposes */
+    id?: string;
+
+    /** ID of associated label */
+    labelId?: string;
+
+    /** ID of associated HelperText */
+    helperTextId?: string;
+
+    /** Attribute indicating that an option with a non-empty string value must be selected */
+    isRequired: boolean;
+
+    /** When passed, will populate the aria-label on the select */
+    ariaLabel?: string;
+
+    /** When true, disables the select */
+    disabled?: boolean;
+
+    /** Populates the selects' options */
+    options: string[];
```

Comment:
I would let the consumer render these as one usually does with a `<select>`, so passed as children. The API would then be:
```
<Select>
  <option>my option</option>
  <option>another!</option>
</Select>
```
which is the same as using regular HTML:
```
<select>
   <option>option 1</option>
</select>
```

This is especially because a select might have an option with a value that is different from its display name:
```
<Select>
  {mediaType.map(mediaType => (
    <option key={mediaType.id} value={mediaType.id}>{mediaType.name}</option>
  )}
</Select>
```